Makefile

```diff
@@ -135,5 +142,5 @@ install-opa:
 
 #--------------------------------------------------------------------
 
-.PHONY: help test fmt fmt-check ci install-opa check \
-  clean-data dummy-config fetch-att fetch-data show-data
+.PHONY: help test coverage quiet-test live-test fmt fmt-check ci clean-data \
+  dummy-config fetch-att show-data fetch-data check install-opa
```

Comment:
I just tidied them up and put them in order. FWIW, we could probably get away with dumping these since IIUC the only impact would be if we had a file or dir with the same name as one of them.